data/society/openstreetmap-places/index.html

```diff
@@ -0,0 +1,118 @@
+---

```

Comment:
**issue**: this should be deleted right?

Reply:
good catch